app/controllers/conversations.js

```diff
@@ -29,55 +30,77 @@ class ConversationsController extends BaseController {
     this.sessionRepository = new SessionRepository(ACTIVE);
   }
 
-  async #notifyAboutConversationCreate(
+  async #notifyAboutConversationEvent(
     ws,
     conversation,
-    currentUserLogin,
-    recipients
+    currentUserParams,
+    recipients,
+    eventType
```

Comment:
let's have `eventType ` as the 1st parameter. This will improve code readability. 

Also, let's create a constant with all event types. e.g.

```
const CONVERSATION_EVENT = {
   CREATE: "create",
   UPDATE: "update",
   DELETE: "delete"
}
```

and then call it 

```
async #notifyAboutConversationEvent(CONVERSATION_EVENT.CREATE, ... )
```